docs-src/go/how-to-define-an-update-type-in-go.md

```diff
@@ -0,0 +1,27 @@
+---
+id: how-to-define-an-update-type-in-go
+title: How to define an Update Type in Go
```

Comment:
I don't love the use of the word `type` here, especially because it is a keyword in Go. I don't think we use the word `type` when talking about signals or queries. Even the Go SDK API is `SetUpdateHandler` not `SetUpdateType`. I would change to `How to define an Update Handler in Go`

Reply:
You are correct - we say "Define Signal" or "How to define a Query" -
I will change this either in this PR or track it and follow up quickly.